bounce/db/membership.py

```diff
@@ -36,62 +48,181 @@ def to_dict(self):
         return {
             'user_id': self.user_id,
             'club_id': self.club_id,
+            'position': self.position,
+            'role': self.role,
             'created_at': self.created_at,
         }
 
 
-def insert(session, club_name, user_id):
+def can_insert(editors_role, members_role):
+    """
+    Determines whether user can insert a memberships to the database
+    Args:
+        editors_role (Role): the role of the member who is deleting the membership
+        members_role (Role): the role of the member who's membership is being deleted
+    """
+
+    if editors_role == 'President':
+        return True
+    # Admin can only insert Member memberships
+    elif editors_role == 'Admin' and members_role == 'Member':
+        return True
+
+
+def can_select(editors_role):
+    # All members can read all memberships
+    if editors_role is None:
+        return False
+    else:
+        return True
+
+
+def can_delete_all(editors_role=None, members_role=None):
+    # Owners can delete all memberships except other presidents
+    if editors_role == 'President' and members_role != 'President':
+        return True
```

Comment:
You forgot the `return False` here.

Reply:
Thanks I'll make that change and add an "else: return False" for all of these.